set.mm

```diff
@@ -325401,29 +325437,29 @@ or are almost disjoint (the interiors are disjoint).  (Contributed by
       $( Lemma for ~ mbfeqa .  (Contributed by Mario Carneiro, 2-Sep-2014.) $)
       mbfeqalem $p |- ( ph ->
         ( ( x e. B |-> C ) e. MblFn <-> ( x e. B |-> D ) e. MblFn ) ) $=
-        ( vy vz wcel wa cr cfv wceq wb syl2anc cmpt ccnv cima cvol cdm cioo crn
-        cv wral cmbf cin cdif cun inundif incom dfin4 eqtri id wss covol cc0 wn
-        cab symdif2 eldifi adantl sylan2 eqid fvmpt2 3eqtr4d ralrimiva nffvmpt1
-        eldif nfeq fveq2 eqeq12d cbvral sylib r19.21bi eleq1d sylan2br anass1rs
-        nfv pm5.32da wfn fmpttd ffnd adantr elpreima syl 3bitr4d con1d syl5eqss
-        ex abssdv unssad sstrd ovolssnul syl3anc nulmbl difmbl syl2anr syl5eqel
-        unssbd unmbl syl5eqelr impbida ralbidv wf ismbf ) ABDEUAZUBLUHZUCZUDUEZ
-        NZLUFUGZUIZBDFUAZUBXLUCZXNNZLXPUIZXKUJNZXRUJNZAXOXTLXPAXOXTAXOOZXSXSXMU
-        KZXSXMULZUMZXNXSXMUNYDYEXNNYFXNNZYGXNNYDYEXMXMXSULZULZXNYEXMXSUKZYJXSXM
-        UOXMXSUPUQXOXOYIXNNZYJXNNAXOURAYIPUSYIUTQVARZYLAYICPAYIYFCAYIYFUMMUHZXM
-        NZYNXSNZSZVBZMVCCMXMXSVDAYRMCAYNCNZYQAYSVBZYQAYTOZYNDNZYNXKQZXLNZOZUUBY
-        NXRQZXLNZOZYOYPUUAUUBUUDUUGAUUBYTUUDUUGSZUUBYTOAYNDCULZNZUUIYNDCVMAUUKO
-        UUCUUFXLAUUCUUFRZMUUJABUHZXKQZUUMXRQZRZBUUJUIUULMUUJUIAUUPBUUJAUUMUUJNZ
-        OZEFUUNUUOIUURUUMDNZEPNZUUNERUUQUUSAUUMDCVEZVFZUUQAUUSUUTUVAJVGBDEPXKXK
-        VHVITUURUUSFPNZUUOFRUVBUUQAUUSUVCUVAKVGBDFPXRXRVHVITVJVKUUPUULBMUUJUUPM
-        WCBUUCUUFBDEYNVLBDFYNVLVNUUMYNRUUNUUCUUOUUFUUMYNXKVOUUMYNXRVOVPVQVRVSVT
-        WAWBWDUUAXKDWEZYOUUESAUVDYTADPXKABDEPJWFZWGWHDYNXLXKWIWJUUAXRDWEZYPUUHS
-        AUVFYTADPXRABDFPKWFZWGWHDYNXLXRWIWJWKWNWLWOWMZWPZGWQAYICUSCPUSZCUTQVARZ
-        YMUVIGHYICWRWSYIWTTZXMYIXAXBXCAYHXOAYFPUSYFUTQVARZYHAYFCPAYIYFCUVHXDZGW
-        QAYFCUSUVJUVKUVMUVNGHYFCWRWSYFWTTZWHYEYFXETXFAXTOZXMYKYIUMZXNXMXSUNUVPY
-        KXNNYLUVQXNNUVPYKXSYFULZXNYKYEUVRXMXSUOXSXMUPUQXTXTYHUVRXNNAXTURUVOXSYF
-        XAXBXCAYLXTUVLWHYKYIXETXFXGXHADPXKXIYBXQSUVELDXKXJWJADPXRXIYCYASUVGLDXR
-        XJWJWK $.
+        ( vy vz wcel wa cdif cr cfv wceq wb cmpt ccnv cv cima cvol cdm cioo crn
```

Comment:
The new proof is longer than the old one: OLD 15 lines + 27 Bytes vs NEW 15 lines + 44 Bytes.  Why should we prefer the longer one?

Reply:
This change was not in the scope of a minimization, but during the cleanup of symdif.

Reply:
So the new symdif in this particular case produces a longer proof, and this cannot be avoided?

Reply:
Maybe it could, but this would mean to restructure the proof of mbfeqalem completely. In this case, the OLD theorem symdif2OLD had to be replaced, and this caused the (slight) enlargement of the proof. 
As already discussed before, to shorten the proofs of high level theorems should not be the primary goal, but clarity and using definitions consequently are also very important (which is the objective here).

Reply:
So far I am not able to do a side-by-side comparison.  I have to finish my work first, before further looking into this.

Reply:
All minimizations are in the second commit [77b1980](https://github.com/metamath/set.mm/pull/2780/commits/77b1980a8ffa8ccef08b0f6c173698cfc8328239) only (if that helps...).

Reply:
I now had a closer look into the proof of mbfeqalem.  In order to avoid a longer proof, you need a companion theorem to difsymdifssd:  (𝜑 → (𝐴 △ 𝐵) ⊆ 𝐶) => (𝜑 → (𝐵 ∖ 𝐴) ⊆ 𝐶).  The current set of theorems are biased towards 𝐴 ∖ 𝐵.  If you need the reverse difference, as in mbfeqalem, then the proofs become more complicated.  This is the background for the observed regression.

I leave it open to further discussion whether one should lift this bias.  For me this issue is resolved.

Reply:
How could we keep @wlammen's information ? By putting it as a TODO in the comment of mbfeqalem ?